gems/sorbet-runtime/lib/types/types/typed_hash.rb

```diff
@@ -3,22 +3,32 @@
 
 module T::Types
   class TypedHash < TypedEnumerable
```

Comment:
I think this type might need to override `build_type` and trigger `keys` and `values` before calling `super`.

Reply:
Yeah indeed, I don't think the order is relevant though?